### PR TITLE
SDI-175 do you want to receive notifications page

### PR DESCRIPTION
**Pull request by steverendell**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,7 +25,7 @@
     "testci": "jest --maxWorkers=2",
     "security_audit": "npx audit-ci --config audit-ci.json",
     "int-test": "cypress run --config video=false",
-    "int-test-ui": "cypress open",
+    "int-test-ui": "cypress open --e2e --browser electron",
```
Comment (petergphillips) [inline package.json:28]: why `electron` here instead of `chrome`?

Comment (petergphillips) [reply package.json:28]: I do like the `e2e` setting though - hadn't realised that we could do that

Comment (steverendell) [reply package.json:28]: I thought it provides a bit of browser variation as we use chrome normally .. although I just remembered the headless run uses electron anyway, so perhaps not! Will change next time.

**Review by petergphillips — APPROVED** [review]
LGTM

**Pull request merged**